fd.go

```diff
@@ -1,4 +1,4 @@
-// +build !windows, !freebsd
+// +build !windows,!freebsd
```

Comment:
Should this also include darwin? It looks like it is just checking procfs.

Reply:
Good question. I guess it compiles on darwin but it probably wouldn't do the right think. Any opposition to me changing `fd.go` to `fd_linux.go` and changing `fd_windows.go` to `fd_noop.go` that just excludes linux?

Reply:
Seems reasonable.